src/platform_impl/ios/monitor.rs

```diff
@@ -35,15 +38,20 @@ impl Drop for VideoMode {
     fn drop(&mut self) {
         unsafe {
             assert_main_thread!("`VideoMode` can only be dropped on the main thread on iOS");
-            msg_send![self.screen_mode, release];
+            msg_send![self.screen_mode, release]
         }
     }
 }
 
 impl VideoMode {
     unsafe fn retained_new(uiscreen: id, screen_mode: id) -> VideoMode {
         assert_main_thread!("`VideoMode` can only be created on the main thread on iOS");
-        let refresh_rate: NSInteger = msg_send![uiscreen, maximumFramesPerSecond];
+        let refresh_rate: NSInteger = if app_state::capabilities().maximum_frames_per_second {
+            msg_send![uiscreen, maximumFramesPerSecond]
+        } else {
+            log::warn!("`-[UIScreen maximumFramesPerSecond]` is unsupported, defaulting to 60 fps");
+            60
```

Comment:
How was this determined to be a reasonable default? (I know the answer, but I want a comment.)

It could also be good to have the warning contain info on both 1) the version the user has, and 2) the version the user would need for this feature.